qiskit_dynamics/pulse/pulse_to_signals.py

```diff
@@ -79,14 +81,60 @@ def __init__(
         self._carriers = carriers or {}
 
     def get_signals(self, schedule: Schedule) -> List[DiscreteSignal]:
-        """
+        r"""Convert a schedule to a corresponding list of DiscreteSignal instances.
+
+        Which channels are converted, and the order they are returned, is controlled by the
+        ``channels`` argument at instantiation. The ``carriers`` instantiation argument sets the
+        analog carrier frequency for each channel, which is fixed for the full duration. For a given
+        channel, the :math:`k^{th}` envelope sample for the corresponding :class:`.DiscreteSignal`
+        is determined according to the following formula:
+
+        .. math::
+            f_k \exp(i(2\pi \Delta\nu(k) k dt + \phi(k) + 2 \pi \phi_a(k))),
+
+        where:
+
+        * :math:`f_k` is the waveform value at the :math:`k^{th}` time step as specified by ``Play``
```

Comment:
Is there a reason `f_k` and `\phi(k)` use different indexing notation? Consider unifying if not.

Reply:
No great reason. I think I maybe did this because of the way I think about them due to internal implementation - the waveform samples (the `f_k`) is stored in an actual array that has an explicit value stored at every time step, whereas I chose a function notation for the phases and frequency difference as they're things that get periodically updated and not explicitly stored at every time step.

Changed `f_k` to `f(k)`.